helms/legion/values.yaml

```diff
@@ -573,6 +573,62 @@ edi:
     # Type: string
     # tlsSecretName: ~
 
+
+# JupyterLab instance
+# It provides web IDE for edition of source codes
+jupyterlab:
+  # JupyterLab could be disabled
+  # Type: bool
+  enabled: true
+
+  # This variable can be uncommented to setup custom image name for JupyterLab instance
+  # Type: string
+  # image: custom-image:1.0
+
+  # Port on which JupyterLab listens income traffic
+  # Type: integer
+  port: 80
+
+  # Resources for each instance
+  # For declaration format see https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+  resources:
+    requests:
+      cpu: "50m"
+      memory: "128Mi"
+    limits:
+      cpu: "100m"
+      memory: "512Mi"
```

Comment:
Please, add a robot test. It should check that the jupiterlab deployment is up and closed by dex.

Reply:
Fixed